CHANGELOG.md

```diff
@@ -2,6 +2,87 @@
 
 All notable changes to this project will be documented in this file.
 
+## Unreleased - as of Sprint 49 end 2016-11-14
+
+### [Added](https://github.com/ManageIQ/manageiq/issues?q=milestone%3A%22Sprint+49+Ending+Nov+14%2C+2016%22+label%3Aenhancement)
+
+- Automate
+  - Service Model
+    - Expose ems_events to Vm service model
+    - Expose a group's filters.
+    - Expose authentication_key in EMS service model
+  - Provisioning
+    - Set zone when deliver a service template provision task
+    - Route VM reconfigure request to appropriate region
```

Comment:
I think original title "Create a request per region for vm reconfigure" makes more sense.